src/Kunstmaan/NodeSearchBundle/Resources/config/update_listener.yml

```diff
@@ -4,7 +4,7 @@ parameters:
 services:
     kunstmaan_node_search.node_index_update.listener:
         class: '%kunstmaan_node_search.node_index_update.listener.class%'
-        arguments: ['@kunstmaan_node_search.search_configuration.node']
+        arguments: ['@kunstmaan_node_search.search_configuration.node', '@doctrine.orm.entity_manager.abstract']
```

Comment:
Inject `@doctrine.orm.entity_manager` or `doctrine.orm.default_entity_manager` instead, as these link to the default orm connection.